datadog/fwprovider/resource_datadog_integration_gcp_sts.go

```diff
@@ -272,6 +282,9 @@ func (r *integrationGcpStsResource) updateState(ctx context.Context, state *inte
 	if hostFilters, ok := attributes.GetHostFiltersOk(); ok && len(*hostFilters) > 0 {
 		state.HostFilters, _ = types.SetValueFrom(ctx, types.StringType, *hostFilters)
 	}
+	if runFilters, ok := attributes.GetCloudRunRevisionFiltersOk(); ok && len(*runFilters) > 0 {
+		state.CloudRunRevisionFilters, _ = types.SetValueFrom(ctx, types.StringType, *runFilters)
```

Comment:
Same: ignoring error

Reply:
That's the intention we verify no errors exist in the diagnostics prior to making this call [here](https://github.com/DataDog/terraform-provider-datadog/blob/b8a14093ba2ef0fcdcfbac3417fc7b0af9898a5e/datadog/fwprovider/resource_datadog_integration_gcp_sts.go#L227-L227)

Reply:
https://github.com/DataDog/terraform-provider-datadog/pull/2612#discussion_r1797400708